docs/reference/ilm/ilm-tutorial.asciidoc

```diff
@@ -7,31 +7,33 @@
 ++++
 
 When you continuously index timestamped documents into {es},
-you typically use a <<data-streams, data stream>> so you can periodically roll over to a
+you typically use a <<data-streams,data stream>> so you can periodically <<index-rollover,roll over>> to a
 new index.
-This enables you to implement a hot-warm-cold architecture to meet your performance
+This enables you to implement a <<data-tiers,hot-warm-cold architecture>> to meet your performance
 requirements for your newest data, control costs over time, enforce retention policies,
 and still get the most out of your data.
 
-TIP: Data streams are best suited for
+TIP: <<data-streams,Data streams>> are best suited for
 <<data-streams-append-only,append-only>> use cases. If you need to update or delete existing time
 series data, you can perform update or delete operations directly on the data stream backing index.
 If you frequently send multiple documents using the same `_id` expecting last-write-wins, you may
-want to use an index alias with a write index instead. You can still use ILM to manage and rollover
+want to use an index alias with a write index instead. You can still use <<index-lifecycle-management,ILM>> to manage and <<index-rollover,rollover>> 
```

Comment:
since we're here

```suggestion
want to use an index alias with a write index instead. You can still use <<index-lifecycle-management,ILM>> to manage and <<index-rollover,roll over>> 
```